opl/opl_sdl.c

```diff
@@ -71,6 +71,7 @@ static uint64_t pause_offset;
 // OPL software emulator structure.
 
 static Chip opl_chip;
+static int opl_new;
```

Comment:
Please add a comment documenting what this variable does, because it's not obvious. I think perhaps "opl_new" is a misleading name.